docs/csharp/language-reference/compiler-options/listed-alphabetically.md

```diff
@@ -59,7 +59,7 @@ The following compiler options are sorted alphabetically. For a categorical list
 |**/incremental**|Enables incremental compilation [obsolete].|  
 |[/keycontainer](../../../csharp/language-reference/compiler-options/keycontainer-compiler-option.md)|Specifies a strong name key container.|  
 |[/keyfile](../../../csharp/language-reference/compiler-options/keyfile-compiler-option.md)|Specifies a strong name key file.|  
-|[/langversion:\<string>](../../../csharp/language-reference/compiler-options/langversion-compiler-option.md)|Specify language version mode: ISO-1, ISO-2, 3, 4, 5, 6, or Default|  
+|[/langversion:\<string>](../../../csharp/language-reference/compiler-options/langversion-compiler-option.md)|Specify language version mode: ISO-1, ISO-2, 3, 4, 5, or Default|  
```

Comment:
Ultimately, as I've stated below, the entire topic needs clarification on which exact csc we're talking about. MSDN has platform versioning information on all it's docs whereas the net docs dont. 

It would be far more straightforward and clear to list supported versions if we could do the same thing, msdn docs go as far back as at least net 1.1 so there's no reason why we shouldn't discuss what the older compilers can/cant do.

(yes I understand that dotnet is a separate entity from ms/msdn now, but you can still view the docs though a microsoft.com address)